common/base/include/claragenomics/utils/cudautils.hpp

```diff
@@ -147,7 +150,7 @@ std::size_t find_largest_contiguous_device_memory_section();
 /// \brief starts an NVTX range for profiling which stops automatically at the end of the scope
 /// \param varname an arbitrary variable name for the nvtx_range object, which doesn't conflict with other variables in the scope
 /// \param label the label/name of the NVTX range
-#define CGA_NVTX_RANGE(varname, label) ::claragenomics::cudautils::nvtx_range varname(label)
+#define CGA_NVTX_RANGE(varname, label) ::genomeworks::cudautils::nvtx_range varname(label)
```

Comment:
This should be `::claraparabricks::genomeworks::cudautils::nvtx_range`

Reply:
good catch! fixed, and added profiling flag to release mode testing